homeassistant/components/analytics/manifest.json

```diff
@@ -5,6 +5,7 @@
   "codeowners": ["@home-assistant/core", "@ludeeus"],
   "dependencies": ["api", "websocket_api"],
   "documentation": "https://www.home-assistant.io/integrations/analytics",
+  "import_executor": true,
```

Comment:
Why is this one added to be in the executor, it has no requirements?

Reply:
Because this one has to pull in lots of other places in core it has a lot of import side effects. Since it's loaded so soon this is effectively where a lot of core components get loaded initially.